.gitignore

```diff
@@ -1,3 +1 @@
-/bin/
-
-.vscode
\ No newline at end of file
+/bin/
```

Comment:
Add back in the newline that was there before, so that there are no changes to this file.